galaxy_ng/app/tasks/deletion.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from pulp_container.app import tasks as pulp_container_tasks
+from pulpcore.app.tasks import orphan_cleanup, reclaim_space
+
+log = logging.getLogger(__name__)
+
+
+def delete_container_distribution(instance_ids):
+    """Deletes a container distribution and push repository related."""
+
+    log.info("Running container.general_multi_delete to delete distro and repo")
+    pulp_container_tasks.general_multi_delete(instance_ids=instance_ids)
+
+    log.info("Running orphan_cleanup to delete CollectionVersion objects and artifacts")
+    orphan_cleanup(content_pks=None, orphan_protection_time=0)
+
+
+def delete_container_image_manifest(repository_pk, content_unit_pks):
+    """Deletes a container image manifest."""
+
+    log.info(f"Running delete manifest for {repository_pk}")
+    pulp_container_tasks.recursive_remove_content(
+        repository_pk=repository_pk,
+        content_units=content_unit_pks,
+    )
+
+    log.info(f"Reclaiming disk space for {repository_pk}")
+    reclaim_space(repo_pks=[repository_pk])
```

Comment:
you need to set the force flag to True https://github.com/pulp/pulpcore/blob/master/pulpcore/app/tasks/reclaim_space.py#L23

Reply:
Thanks @ipanova 💯